package.json

```diff
@@ -47,5 +47,8 @@
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-prettier": "^4.2.1",
     "prettier": "^2.8.8"
+  },
+  "dependencies": {
```

Comment:
I believe this doesn't need to be installed for all the packages since it's only present in the `ui`. I'd suggest to add it to the `packages/ui` deps rather than the global one.